recipes/orc/build.sh

```diff
@@ -0,0 +1,47 @@
+mkdir -p build
+cd build
+
+declare -a _CMAKE_EXTRA_CONFIG
+if [[ ${HOST} =~ .*darwin.* ]]; then
+    if [[ ${_XCODE_BUILD} == yes ]]; then
+        _CMAKE_EXTRA_CONFIG+=(-G'Xcode')
+        _CMAKE_EXTRA_CONFIG+=(-DCMAKE_OSX_ARCHITECTURES=x86_64)
+        _CMAKE_EXTRA_CONFIG+=(-DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT})
+        _VERBOSE=""
+    fi
+    unset MACOSX_DEPLOYMENT_TARGET
+    export MACOSX_DEPLOYMENT_TARGET
+    _CMAKE_EXTRA_CONFIG+=(-DCMAKE_AR=${AR})
+    _CMAKE_EXTRA_CONFIG+=(-DCMAKE_RANLIB=${RANLIB})
+    _CMAKE_EXTRA_CONFIG+=(-DCMAKE_LINKER=${LD})
+fi
```

Comment:
Why are these flags necessary? (and also for linux)

Reply:
I don't think they are necessary for builds in `conda-forge`. They are coming in from the original recipe from Anaconda and I wanted to stay as close as possible to it. @msarahan Can you give insight on this?

Reply:
Cmake is terrible with non-system sysroots. You can try to remove this, but I think it is probably necessary for our compilers to work correctly. Since conda forge is using it complete now also, it's probably important here the same as it is on anaconda recipes.

Reply:
Hi @msarahan , thanks for the explanation! :)